pyanaconda/modules/payload/live/live_image_interface.py

```diff
@@ -126,7 +126,6 @@ def PostInstallWithTask(self) -> ObjPath:
 
         * [TODO] copy driver disk files (Payload)
         * [NO] check installation requirements were applied (Payload)
-        * Update BLS entries (LiveOSPayload)
```

Comment:
Wasn't this removed by accident?

Reply:
Nope. I'm using the BLS entries task in the LiveOS payload in this PR.

Reply:
Yes but it is still done in the LiveImage payload task, right? So I'd just remove LiveOSPayload reference, but it is not important, let's leave it as it is.